examples/python-ml-experiment-tracker/tracker/cli.py

```
@@ -0,0 +1,340 @@
+        except AuthError as e:
+            err_console.print(f"[bold red]Auth error:[/] {e}")
+            raise SystemExit(1)
+        except APIError as e:
+            err_console.print(f"[bold red]API error {e.status_code}:[/] {e.message}")
+            raise SystemExit(1)
+        except httpx.ConnectError:
+            cfg = load_config()
+            err_console.print(
+                f"[bold red]Connection refused.[/] Is InsForge running at {cfg.server_url}?"
+            )
+            raise SystemExit(1)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)